### PR TITLE
Syndicate softsuits Update

### PR DESCRIPTION
**Pull request by shazbot194**
## About The Pull Request

This updates the decade-old syndicate softsuit sprite to something more in line with Eris's art style, complete with new sprites for mob, icons, and even in hands. It also adds a black-market versions for uplinks.

image

## Why It's Good For The Game

It updates old sprites.

## Changelog
:cl: Shazbot194
add: Added new sprites for syndicate softsuits, as well as an uplink version.
/:cl:

### CODE REVIEW

**Comment by shazbot194** [comment]
Man, github and DM are amazing, I only updated them like a day ago

**Comment by shazbot194** [comment]
Due to requests from other devs, I have removed all black versions of the suits to reduce the total amount.

**Comment by shazbot194** [comment]
How? I just fixed the conflicts?!?!?

**Comment by Iglpigl** [comment]
Doesn't pass Travis.

**Comment by shazbot194** [comment]
Hmm, it totally should, it compiles and runs just fine.

**Comment by zombiemimic** [comment]
does it get any warnings from DM?

**Comment by shazbot194** [comment]
None, one moment

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 1]
```diff
@@ -21,13 +21,15 @@
 /obj/item/clothing/head/space/emergency
 	name = "Emergency Space Helmet"
```
Comment (Firefox13) [inline code/modules/clothing/spacesuits/miscellaneous.dm:22]:
```suggestion
	name = "emergency space helmet"
```

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**